.travis.yml

```diff
@@ -2,10 +2,9 @@ sudo: false
 language: php
 
 php:
-  - 7.1
```

Comment:
We're compatible with 7.1 per composer.json, so I think it should also be tested here still.

Reply:
What do you mean? https://github.com/nikic/FastRoute/pull/201/commits/195664b1f892b9a95478aef54ae380f920544404#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R25

Reply:
Oops, I missed that, sorry!

IMHO good targets for open-source libraries are PHP 7.1 or PHP 7.4, and versions in between are not really worthwhile. Assuming it's still a bit early for PHP 7.4, I'd prefer to keep PHP 7.1 support here.

Reply:
That restrict us regarding dev tools ([PHPUnit](https://packagist.org/packages/phpunit/phpunit) and [Doctrine CS](https://packagist.org/packages/doctrine/coding-standard)), tough.

We're dropping 7.1 in Doctrine and am updating every lib of mine to use 7.4 so I consider it pretty much dead :stuck_out_tongue: 

Do you mind if we use `^7.5 || ^8.5` for PHPUnit and `^6.0 || ^7.0` for Doctrine CS (and validate the CS using PHP 7.2)?

Reply:
php7.1 not supported 
https://www.php.net/supported-versions.php

Reply:
Specifying multiple versions via `||` is definitely fine and I don't see a problem with validating CS on only 7.2 (or heck 7.4) either.

Reply:
Can't do PHP 7.4 on CS, otherwise it will complain about properties that are not declaring types when they could :smile:

Reply:
Commit dropped and added multiple versions :+1: